dataservice/api/read_group/models.py

```diff
@@ -27,9 +27,14 @@ class ReadGroup(db.Model, Base):
     quality_scale = db.Column(db.Text(),
                               doc='The scale used to encode quality scores')
 
-    genomic_files = db.relationship('GenomicFile',
-                                    secondary='read_group_genomic_file',
-                                    backref=db.backref('read_groups'))
+    genomic_files = association_proxy(
+        'read_group_genomic_files', 'genomic_file',
+        creator=lambda genomic_file:
+        ReadGroupGenomicFile(genomic_file=genomic_file))
```

Comment:
Kinda difficult to tell that this is actually the body of the lambda. Indent or move up a line?